crates/neon-sys/native/src/neon.cc

```diff
@@ -221,15 +221,21 @@ extern "C" bool Neon_ArrayBuffer_New(v8::Local<v8::ArrayBuffer> *out, v8::Isolat
 
 
 extern "C" size_t Neon_ArrayBuffer_Data(v8::Isolate *isolate, void **base_out, v8::Local<v8::ArrayBuffer> buffer) {
-#if (V8_MAJOR_VERSION >= 8)
-    auto contents = buffer->GetBackingStore();
-    *base_out = contents->Data();
-    return contents->ByteLength();
-#else
-    v8::ArrayBuffer::Contents contents = buffer->GetContents();
-    *base_out = contents.Data();
-    return contents.ByteLength();
-#endif
+  #if _MSC_VER && NODE_RUNTIME_ELECTRON && NODE_MODULE_VERSION >= 89
+    v8::Local<v8::Object> local;
+    node::Buffer::New(isolate, buffer, 0, buffer->ByteLength()).ToLocal(&local);
```

Comment:
I like this hack, I think it quietly fixes a painful build issue so users don't need to worry about it. I'd just add some comments explaining why it's needed, for internal maintainability.